### DISCUSSION TITLE
Speech Recognition,

### DISCUSSION THREAD

**Discussion by Moukarnold** [Moukarnold/capstone-project#15] [Announcements]
did someone already work with Speech Recognition library? i recieve this error "ReferenceError: regeneratorRuntime is not defined"
with the react-speech-recognition

**Comment by MerleMak** [comment]
Are you using the Web Speech API?

**Reply by Moukarnold** [reply]
no

**Reply by Moukarnold** [reply]
this one https://www.npmjs.com/package/react-speech-recognition

**Comment by MerleMak** [comment]
please commit your current code and send a PR

**Reply by Moukarnold** [reply]
https://github.com/Moukarnold/capstone-project/pull/14

**Comment by MerleMak** [comment]
please add this import `import "regenerator-runtime/runtime";` to your _app.js and see if that already fixes the problem :)

**Reply by Moukarnold** [reply]
doesn´t work

**Reply by Moukarnold** [reply]
it´s better now, thank you
